diff --git a/config.def.h b/config.def.h
index 91ab8ca..e3b469b 100644
--- a/config.def.h
+++ b/config.def.h
@@ -201,6 +201,8 @@ static Shortcut shortcuts[] = {
        { TERMMOD,              XK_Y,           selpaste,       {.i =  0} },
        { ShiftMask,            XK_Insert,      selpaste,       {.i =  0} },
        { TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
+       { ShiftMask,            XK_Page_Up,     kscrollup,      {.i = -1} },
+       { ShiftMask,            XK_Page_Down,   kscrolldown,    {.i = -1} },
 };

